apps/storybook/src/forms/Checkbox.stories.tsx

```diff
@@ -0,0 +1,95 @@
+import type { Meta, StoryObj } from "@storybook/react";
+
+import { Checkbox, Flex, Text } from "@optiaxiom/react";
+
+const meta: Meta<typeof Checkbox> = {
+  component: Checkbox,
+};
+
+export default meta;
+
+type Story = StoryObj<typeof Checkbox>;
+
+export const Basic: Story = {
+  args: {
+    children: "Label",
+  },
+};
+export const HelperText: Story = {
+  args: {
+    children: (
+      <Text>
+        Label
+        <Text as="p" color="fg.secondary" fontSize="sm">
+          Helper Text
+        </Text>
+      </Text>
```

Comment:
children is already wrapped in text - so let's use fragment here instead
```suggestion
      <>
        Label
        <Text as="p" color="fg.secondary" fontSize="sm">
          Helper Text
        </Text>
      </>
```

Reply:
addressed bhai.
Just exposing endDecorator for this Component